### ISSUE TITLE
JVM Segmentation error when building the database from batch image

### ISSUE THREAD

**Issue by ourboy** [merative/spm-kubernetes#51]
Since the 12th of December 2020, a JVM Segmentation error when building the database from batch image. 

prior to the 12th of December this was not an issue.

see error below
```
dispmsg:
     [echo] 10:48:54 Starting batchlauncher
[batchlauncher] Using configured properties for logging.
[batchlauncher] Running a Single Batch Program.
[batchlauncher] Connecting to DB2 data source : com.ibm.db2.jcc.DB2SimpleDataSource.
[batchlauncher] 'batch.username' not found.
[batchlauncher] Batch invoking : 'curam.util.internal.userpreference.intf.UserPreferenceLoader.insertUserPreferencesToDatabase'.
[batchlauncher] Unhandled exception
[batchlauncher] Type=Segmentation error vmState=0x00080002
[batchlauncher] J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
[batchlauncher] Handler1=00007F4165F26CA0 Handler2=00007F416580B140 InaccessibleAddress=00007F41AE96D817
[batchlauncher] RDI=00007F4160015CA0 RSI=00007F41AE96D817 RAX=0000000000000000 RBX=00007F41AE96D817
[batchlauncher] RCX=00007F4165FEBD98 RDX=00007F41601F1450 R8=00007F415F606840 R9=000000000000000A
[batchlauncher] R10=0000000000000001 R11=0000000000000000 R12=0000000000000005 R13=0000000000000041
[batchlauncher] R14=00007F41AE96D817 R15=00000000000000A8
[batchlauncher] RIP=00007F4165FB4C26 GS=0000 FS=0000 RSP=00007F4166E61460
[batchlauncher] EFlags=0000000000010202 CS=0033 RBP=00007F4166E61650 ERR=0000000000000004
[batchlauncher] TRAPNO=000000000000000E OLDMASK=0000000000000000 CR2=00007F41AE96D817
[batchlauncher] xmm0 0000000000000000 (f: 0.000000, d: 0.000000e+00)
[batchlauncher] xmm1 534e656c62616e65 (f: 1650552448.000000, d: 1.981380e+93)
[batchlauncher] xmm2 ffffffffffffffff (f: 4294967296.000000, d: -nan)
[batchlauncher] xmm3 120c00b42a1100c6 (f: 705757376.000000, d: 9.683534e-222)
[batchlauncher] xmm4 011700b92b2e01a7 (f: 724435392.000000, d: 2.096455e-303)
[batchlauncher] xmm5 0500990b00b61f00 (f: 11935488.000000, d: 1.395228e-284)
[batchlauncher] xmm6 c60c00b42a1800c6 (f: 706216128.000000, d: -2.773258e+29)
[batchlauncher] xmm7 00b42ab02b050099 (f: 721748096.000000, d: 2.871841e-305)
[batchlauncher] xmm8 2a1100c60c00b42a (f: 201372720.000000, d: 4.633484e-106)
[batchlauncher] xmm9 ffff00ff0000ffff (f: 65535.000000, d: -nan)
[batchlauncher] xmm10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
[batchlauncher] xmm11 000000004d570a3d (f: 1297549824.000000, d: 6.410748e-315)
[batchlauncher] xmm12 000000004a09a025 (f: 1242144768.000000, d: 6.137011e-315)
[batchlauncher] xmm13 000000004b2c0833 (f: 1261176832.000000, d: 6.231042e-315)
[batchlauncher] xmm14 000000004be50e00 (f: 1273302528.000000, d: 6.290950e-315)
[batchlauncher] xmm15 000000004a373e68 (f: 1245134464.000000, d: 6.151782e-315)
[batchlauncher] Module=/opt/ibm/java/jre/lib/amd64/compressedrefs/libj9vm29.so
[batchlauncher] Module_base_address=00007F4165E91000
[batchlauncher] Target=2_90_20201102_458768 (Linux 4.4.0-148-generic)
[batchlauncher] CPU=amd64 (48 logical CPUs) (0x3ef3728000 RAM)
[batchlauncher] ----------- Stack Backtrace -----------
[batchlauncher] (0x00007F4165FB4C26 [libj9vm29.so+0x123c26])
[batchlauncher] (0x00007F4165FB54D3 [libj9vm29.so+0x1244d3])
[batchlauncher] (0x00007F4165F9ACAF [libj9vm29.so+0x109caf])
[batchlauncher] (0x00007F4165FB6BF1 [libj9vm29.so+0x125bf1])
[batchlauncher] (0x00007F4165F9184F [libj9vm29.so+0x10084f])
[batchlauncher] (0x00007F4165F984C6 [libj9vm29.so+0x1074c6])
[batchlauncher] (0x00007F4165F8D8CA [libj9vm29.so+0xfc8ca])
[batchlauncher] (0x00007F4165F8EFC0 [libj9vm29.so+0xfdfc0])
[batchlauncher] (0x00007F4165F8F689 [libj9vm29.so+0xfe689])
[batchlauncher] (0x00007F4165F8FDBE [libj9vm29.so+0xfedbe])
[batchlauncher] (0x00007F4165F84788 [libj9vm29.so+0xf3788])
[batchlauncher] (0x00007F4165F859D6 [libj9vm29.so+0xf49d6])
[batchlauncher] (0x00007F4165F16607 [libj9vm29.so+0x85607])
[batchlauncher] (0x00007F4165F17BBB [libj9vm29.so+0x86bbb])
[batchlauncher] (0x00007F4165F19E01 [libj9vm29.so+0x88e01])
[batchlauncher] (0x00007F4165EA809F [libj9vm29.so+0x1709f])
[batchlauncher] (0x00007F4165EA3C50 [libj9vm29.so+0x12c50])
[batchlauncher] (0x00007F4165F61DB2 [libj9vm29.so+0xd0db2])
[batchlauncher] ---------------------------------------
[batchlauncher] JVMDUMP039I Processing dump event "gpf", detail "" at 2020/12/14 10:49:02 - please wait.
[batchlauncher] JVMDUMP032I JVM requested System dump using '/opt/ibm/Curam/release/buildlogs/core.20201214.231413.421.0001.dmp' in response to an event
[batchlauncher] JVMDUMP010I System dump written to /opt/ibm/Curam/release/buildlogs/core.20201214.231413.421.0001.dmp
[batchlauncher] JVMDUMP032I JVM requested Java dump using '/opt/ibm/Curam/release/buildlogs/javacore.20201214.231413.421.0002.txt' in response to an event
[batchlauncher] JVMDUMP010I Java dump written to /opt/ibm/Curam/release/buildlogs/javacore.20201214.231413.421.0002.txt
[batchlauncher] JVMDUMP032I JVM requested Snap dump using '/opt/ibm/Curam/release/buildlogs/Snap.20201214.231413.421.0003.trc' in response to an event
[batchlauncher] JVMDUMP010I Snap dump written to /opt/ibm/Curam/release/buildlogs/Snap.20201214.231413.421.0003.trc 
[batchlauncher] JVMDUMP032I JVM requested JIT dump using '/opt/ibm/Curam/release/buildlogs/jitdump.20201214.231413.421.0004.dmp' in response to an event
```

**Comment by ourboy** [comment]
[javacore.20201214.231413.421.0002.txt](https://github.com/IBM/spm-kubernetes/files/5703570/javacore.20201214.231413.421.0002.txt)

**Comment by gillesde** [comment]
Raised issues:
-  **ibmruntimes/ci.docker**  whom provides the `ibmcom/websphere-liberty:20.0.0.9-full-java8-ibmjava-ubi` image: https://github.com/ibmruntimes/ci.docker/issues/104
-  **eclipse/openj9** whom provides the `java version 1.8.0_271` used by the images: https://github.com/eclipse/openj9/issues/11504

**Comment by andreyzher** [comment]
A fix for the underlying issue was included in the [0.24.0 release](https://github.com/eclipse/openj9/releases/tag/openj9-0.24.0) of OpenJ9, but this would still have to come up through IBM Java SDK take-on/release process.

**Comment by wwwild** [comment]
The underlying issue ([11504](https://github.com/eclipse-openj9/openj9/issues/11504)) has been closed. Closing this issue.
